### PR TITLE
Add instructions for setting up GH npm token

### PR DESCRIPTION
**Pull request by hardillb**
Instructions for running devices with Staging.

### CODE REVIEW

File: development/staging.md [hunk 1]
```diff
@@ -34,3 +34,33 @@ Currently there is no auto deployment to staging, this should be rectified in th
...
 When setting up a team you'll need to enter billing details. For credit card
 details, use [the Stripe mock data](https://stripe.com/docs/testing#testing-interactively).
+
+## Using the FlowForge Device Agent with staging
+
+Staging uses pre-release npm packages stored in a GitHut npm repository. To be able to use these packages you will need to authenticate with the repository.
+
+You will need to create a GH Personal token
+
+1. Go to your account -> settings (click on avatar then near bottom of menu)
```
Comment (ZJvandeWeg) [inline development/staging.md:44]: We can remove 3 bullet points completely:
```suggestion
1. Go to your [classic personal access tokens](https://github.com/settings/tokens)
```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

**Comment by ZJvandeWeg** [comment]
When updated, feel free to merge. Also; Thanks! 👍

File: development/staging.md [hunk 2]
```diff
@@ -34,3 +34,33 @@ Currently there is no auto deployment to staging, this should be rectified in th
 
 When setting up a team you'll need to enter billing details. For credit card
 details, use [the Stripe mock data](https://stripe.com/docs/testing#testing-interactively).
+
+## Using the FlowForge Device Agent with staging
+
+Staging uses pre-release npm packages stored in a GitHut npm repository. To be able to use these packages you will need to authenticate with the repository.
```
Comment (hardillb) [inline development/staging.md:40]:
```suggestion
Staging uses pre-release npm packages stored in a GitHub npm repository. To be able to use these packages you will need to authenticate with the repository.
```

**Comment by hardillb** [comment]
Changes made, needs an accepted review to allow merging

**Pull request merged**